test/e2e/apimachinery/table_conversion.go

```diff
@@ -110,9 +110,9 @@ var _ = SIGDescribe("Servers with support for Table transformation", func() {
 			Do().Into(pagedTable)
 		framework.ExpectNoError(err, "failed to get pod templates in Table form in namespace: %s", ns)
 		gomega.Expect(len(pagedTable.Rows)).To(gomega.Equal(2))
```

Comment:
Can we merge https://github.com/kubernetes/kubernetes/pull/79685 first before this?

Reply:
Sure :)